source/_components/somfy.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: page
+title: "Somfy"
+description: "Instructions on how to set up the Somfy hub within Home Assistant."
+date: 2019-06-07 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category:
+  - Hub
+ha_iot_class: Cloud Polling
+ha_release: 0.95
+ha_qa_scale: gold
```

Comment:
The quality scale index is incorrect IMHO. For a gold standard, there should be tests between HA & the integration. Currently, I only see tests for the configuration flow. (which are excluded from the code coverage, which it should have been btw...)

See: https://developers.home-assistant.io/docs/en/integration_quality_scale_index.html

For now, I suggest removing this.

Reply:
Are you talking about this: 
>Tests for reading data from/controlling the integration (docs)

What do you think to replace by Silver :2nd_place_medal: ?

Reply:
For the coverage, I will see how include my tests.

Reply:
Well for silver it should:

✅  Connection/configuration is handled via a component.
❌  Set an appropriate SCAN_INTERVAL (if a polling integration)
❌ Raise PlatformNotReady if unable to connect during platform setup (if appropriate)
❓ Handles expiration of auth credentials. Refresh if possible or print correct error and fail setup. ❓ If based on a config entry, should trigger a new config entry flow to re-authorize.
❓ Handles internet unavailable. Log a warning once when unavailable, log once when reconnected.
❓ Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.
❌ Set available property to False if appropriate (docs)
✅ Entities have unique ID (if available) (docs)

Even though I haven't checked on the question marks,  I still see missing things for a silver 🥈

Reply:
You are right. I will remove it. I have still a lot of stuff to do it seems. 
Sadly, it seems that existing component with qa scale don't respect these rules. Do you know a good one?

Reply:
The QA scale should be correct in the documentation. If you feel like one is incorrect, feel free to raise an issue or open a PR.

Hue & deCONZ are probably good examples.

Reply:
Thank you for the information.